src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -236,5 +361,106 @@ protected override bool ValidateConfiguration(DiscordNotificationSettings settin
             };
             await Send(notification, settings);
         }
+
+        private DiscordEmbed createDiscordEmbed(DiscordAuthor author, MovieRequests req, string imageUrl, bool compactEmbed, bool mentionUser)
+        {
+            author.name = $"{author.name} 🎬";
+            DiscordImage image = null;
+            DiscordImage thumbnail = null;
+            if (compactEmbed)
+                thumbnail = new DiscordImage { url = imageUrl };
+            else
+                image = new DiscordImage { url = imageUrl };
+
+            string description = null;
+            if (!compactEmbed)
+                description = MovieRequest.Overview;
+
+            List<DiscordField> fields = new List<DiscordField>();
+
+            string alias = MovieRequest.RequestedUser.Alias;
+            if (UsingAliasAsMention && mentionUser)
+            {
+                fields.Add
+                (
+                    new DiscordField
+                    {
+                        name = "Honourable Mentions",
+                        value = alias
+                    }
+                );
+                alias = MovieRequest.RequestedUser.UserName;
+            }
+            if (UsingAliasAsMention || !alias.HasValue())
+                alias = req.RequestedUser.UserName;
+
+            DiscordFooter footer = new DiscordFooter
+            {
+                text = $"Requested by {alias}  on {MovieRequest.RequestedDate.ToLongDateString()}"
+            };
+
+            DiscordEmbed embed = new DiscordEmbed
+            {
+                author = author,
+                title = $"{MovieRequest.Title} ({MovieRequest.ReleaseDate.Year})",
+                url = $"{IMDB_BASE_URL}{MovieRequest.ImdbId}",
+                thumbnail = thumbnail,
+                image = image,
+                description = description,
+                footer = footer,
+                fields = fields
+            };
+            return embed;
+        }
+
+        private DiscordEmbed createDiscordEmbed(DiscordAuthor author, ChildRequests req, string imageUrl, bool compactEmbed, bool mentionUser)
```

Comment:
Since this is essentially the same as the method above but we are passing in slightly different objects, we should make a common object and populate that then pass it in. 

This way we can have 1 method. 

The new class would have the release date, overview, title properties etc.